caps.cabal

```diff
@@ -11,7 +11,7 @@ cabal-version:       >=1.10
 
 library
   exposed-modules:     Monad.Capabilities
-  build-depends:       base >=4.10 && <4.16,
+  build-depends:       base >=4.10 && <=4.17,
```

Comment:
1. Please stick to the practice of using non-inclusive upper bounds, i.e. `<4.18` instead of `<=4.17`
2. Update CI configuration to test the new `base` versions that are now allowed
3. Maybe add support for 4.18 too? Would it be hard?